hack/validate-ci-build-clusters.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+workdir="$( mktemp -d )"
+trap 'rm -rf "${workdir}"' EXIT
+
+base_dir="${1:-}"
+
+if [[ ! -d "${base_dir}" ]]; then
+  echo "Expected a single argument: a path to a directory with release repo layout"
+  exit 1
+fi
+
+cp -r "${base_dir}/"* "${workdir}"
```

Comment:
nit: we could work on the current dir, and `git diff` on it.
See an example: https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml#L1113-L1126

We need to add git into the image if you want to go that way.

Reply:
Probably fine as is

Reply:
Heh, I was going to suggest the same in the original PR, but after thinking about it a bit, we probably do this to make it easier to execute these tests in a local clone without disturbing the working directory.